File: python/1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hash_map = {}
        for i, num in enumerate(nums):
            complement = target - num
            if complement in hash_map:
                return [hash_map[complement], i]
            hash_map[num] = i

File: C#/1.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        Dictionary<int, int> dict = new Dictionary<int, int>();
        for(int i = 0; i < nums.Length; i++){
            int complement = target - nums[i];
            if(dict.ContainsKey(complement)){
                return new int[] { dict[complement], i };
            }
            dict[nums[i]] = i;
        }
        return new int[] { };
    }
}